asyncssh/connection.py

```diff
@@ -179,6 +179,7 @@
 _ProtocolFactory = Union[_ClientFactory, _ServerFactory]
 
 _Conn = TypeVar('_Conn', 'SSHClientConnection', 'SSHServerConnection')
+_ConnSelf = TypeVar("_ConnSelf", bound="SSHConnection")
```

Comment:
Apologies for the slow response. Somehow I missed the notification on this.

Would you might changing this to use single quotes? That's the convention in AsyncSSH.